user-docs/modules/ROOT/pages/sop-branching-fedora-iot.adoc

```diff
@@ -0,0 +1,189 @@
+= Branching Fedora IoT
+
+Branched is the name given to a version of Fedora that has "branched" from the rolling Rawhide tree and will become the next stable Fedora release. This document will detail the steps for branching Fedora IoT from Rawhide. 
+
+To complete these steps and open a pull request you should only need a valid [Fedora account](https://docs.fedoraproject.org/en-US/fedora-accounts/user/).
```

Comment:
Should likely read "and open any pull requests you need" as there's more than one needed.

Reply:
Added.